tests/functional/login.js

```diff
@@ -0,0 +1,80 @@
+const Nightmare = require('nightmare');
+const assert = require('chai').assert;
+
+const url = 'http://localhost:3333/';
+
+
+
+describe('Login tests', function() {
+
+  this.timeout('30s');
+
+  let browser = null;
+
+  beforeEach( () => {
+    browser = new Nightmare({ 
+      // to make the typed input much faster
+      typeInterval: 20,
+      // uncomment this to show the browser and the debug window
+      // openDevTools: {
+      //   mode: "detach"
```

Comment:
It would be great to have the script take in a parameter which could be used to change this setting. The default should probably be a hidden browser window and it should be shown only if requested.

Reply:
Well, then the bash script should parse them, add them to the testing script which is implicitly invoked with yarn, then I need to parse the command again in javascript.

So I think WAY to much work (and hassle for me) for a small feature in the testrunner. But if you want it you can make a PR of course :smile: